### PR TITLE
Add Aircoins ( AIRx ) Logo ( Bep20 )

### PR DESCRIPTION
**Pull request by Shyamthakur2**

### CODE REVIEW

**Comment by Shyamthakur2** [comment]
@zachzwei @catenocrypt @Iamdeadlyz 
Please update the logo, fee already has been done. Thanks

File: blockchains/smartchain/assets/0x6Fb05b156788E88c8ad1e057e729362ff8c39d93/info.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Aircoins",
+  "website": "https://www.aircoins.io/",
+  "coinmarketcap": "https://coinmarketcap.com/currencies/aircoins",
+  "coingecko": "https://www.coingecko.com/en/coins/aircoins",
+  "reddit": "https://www.reddit.com/r/Aircoins",
+  "telegram": "https://t.me/aircoins",
```
Comment (vikmeup) [inline blockchains/smartchain/assets/0x6Fb05b156788E88c8ad1e057e729362ff8c39d93/info.json:7]: Can you delete unnecessary field? That are not required by our format

**Review by vikmeup — CHANGES_REQUESTED** [review]

File: blockchains/smartchain/assets/0x6Fb05b156788E88c8ad1e057e729362ff8c39d93/info.json (hunk 1 above)
Comment (Shyamthakur33) [reply blockchains/smartchain/assets/0x6Fb05b156788E88c8ad1e057e729362ff8c39d93/info.json:7]: > Can you delete unnecessary field? That are not required by our format

I am unable to edit, it's saying to delete all file, could you plz give favor to do it?

Comment (Shyamthakur2) [reply blockchains/smartchain/assets/0x6Fb05b156788E88c8ad1e057e729362ff8c39d93/info.json]: @vikmeup it has been fixed, Please check, thanks
Hope, you also will fetch token value with cmc and cg

**Pull request merged**